tests/tablespaces/Makefile

```diff
@@ -0,0 +1,49 @@
+PGVERSION ?= 10
+
+build:
+	$(MAKE) -C ../../ build-test-pg$(PGVERSION)
+	docker-compose build --build-arg "PGVERSION=$(PGVERSION)"
+
+teardown:
+	docker-compose down --volumes --remove-orphans
+
+tail:
+	docker-compose logs -f
+
+state:
+	docker-compose exec monitor pg_autoctl show state
+
+watch:
+	docker-compose exec monitor pg_autoctl watch
+
+test_01:
+	docker-compose up -d node1
+	TEST_NUM="01" docker-compose up --exit-code-from=test test
+
+test_02:
+	docker-compose up -d node2
+	TEST_NUM="02" docker-compose up --exit-code-from=test test
+
+test_03:
+	# Pollute the data dir so pg_rewind fails
+	docker-compose exec node2 mkdir '/var/lib/postgres/data/base/1/99999999'
+	docker-compose exec monitor pg_autoctl perform failover
+	TEST_NUM="03" docker-compose up --exit-code-from=test test
+
+test_04:
+	docker-compose pause node1
```

Comment:
This requires a comment too, as we would usually stop the pg_autoctl process from the Python test file. I'm not sure if we can do that when using docker-compose, as I suppose it would also stop the container entirely. Comparing the two approaches in a comment here looks like a good idea.